client/src/app/services/api.service.ts

```diff
@@ -24,27 +24,27 @@ export class StaticAPIService {
     return this.httpClient.get(`${this.API_URL}/students/`);
   }
 
-  getStudentsfromTurma(collaboratorID: String, turmaID: String) {
-    return this.httpClient.get(`${this.API_URL}/chamada/` + collaboratorID + '/' + turmaID)
+  getStudentsFromClassGroup(collaboratorID: String, classGroupId: String) {
+    return this.httpClient.get(`${this.API_URL}/attendance/` + collaboratorID + '/' + classGroupId);
```

Comment:
Be careful with these URLs. We should create a file with endpoint constants and call endpoints only through these constants. It greatly improves maintainability and we absolutely will benefit from it (:

Reply:
Never mind, just realized that this is exactly what I was talking about.